### ISSUE TITLE
Syntax Error in Joint Groups

### ISSUE THREAD

**Issue by Sabel215** [kergalym/YABEE#30]
When loading a model with an armature exported with this Yabee version, I encountered this syntax error:

```
Error in /c/Users/<OMITTED FOR PRIVACY>/model62.egg at line 123, column 64:
        <Scalar> membership { 0.018510 }  <Ref> { Character Mesh }
                                                               ^
syntax error
```
The line it's referencing is in this context:
```
<Joint> Root {
      <Transform> {
        <Matrix4> {
          1.0 1.1763258100927487e-07 4.6209959236875875e-07 0.0 
          1.4210854715202004e-14 -0.9690935015678406 0.24669349193572998 0.0 
          4.7683673187748354e-07 -0.24669364094734192 -0.9690931439399719 0.0 
          3.276144107222832e-17 0.5618367195129395 3.8937995433807373 1.0 
        }
      }
      <VertexRef> {
        0 2132 5663 5676 9476 11608 15137 15152
        <Scalar> membership { 0.018510 }  <Ref> { Character Mesh }
      }
```
I believe that line is assigning vertex weights to that joint, but I could be wrong. In any case, I was able to fix the error by running a Find and Replace to change all instances of
`{ Character Mesh }`
to
`{ "Character Mesh" }`

Knowing that, I went into the egg_writer.py file from this project and found the line of code that writes out that <Scalar> membership tag. It's on line 364.
`vref_str += '  <Ref> { %s }\n}\n' % vpool`
I added quotation marks around the %s tag:
`vref_str += '  <Ref> { "%s" }\n}\n' % vpool`
And Yabee is now writing the egg file with quotation marks around `Character Mesh` on those lines.
This fixed the problem, but I don't know if there's anything more systemic going on that could cause the problem to appear in other places, or if that fix is in keeping with your coding style. Whether you approve of my fix or not, I hope it helps.

**Comment by Sabel215** [comment]
Additionally, I'm interested in helping with this project. If you might like to bring me on board and want to talk about it, you can message me on the Panda3D forums by searching for @Dove.

**Comment by kergalym** [comment]
Can you provide your .blend file, please?
The lines you're mentioned aren't changed by me yet, but I'll study why YABEE changes it's behavior. 
I think this bug happens due to changes which Python 3x bring,

**Comment by kergalym** [comment]
These lines don't include white space escape, any spaces in the mesh name should be replaced with underscore sign by user or by YABEE while exporting.

**Comment by kergalym** [comment]
Fixed in [ad99ed2](https://github.com/kergalym/YABEE/commit/ad99ed2d9c686badeb1bb92e33a5ec9daeddb536).

**Comment by kergalym** [comment]
@Sabel215  About coding style: it was coding style which the author of original YABEE (which is designed for Blender 2.5, 2.6, 2.7) preferred. Unfortunately, I have not so much free time to do full code refactoring.
